src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+        <head>
+                <title>CodeU Chat App</title>
+                <link rel="stylesheet" href"/css/main.css">
+        </head>
+        <body>
+                <nav>
+                  <a id="navTitle" href"/"CodeU Chat App</a>
+                  <a href"/conversations">Conversations</a>
+                  <% (if(request.getSession().getAttribute("user") != null) { %>
+                    <a>Hello <%=request.getSession().getAttribute("user") %>!</$
+		    <%if(request.getSession().getAttribute("user").equals("EmilyArroyo"))|if(request.getSession().getAttribute("user").equals("AlexandriaStorm"))if(request.getSession().getAttribute("user").equals("AnaVitoriadoValleCosta"))|if(request.getSession().getAttribute("user").equals("KevinWorkman"))|if(request.getSession().getAttribute("user").equals("GavinLifrieri")) { %>
```

Comment:
I'd recommend trying to format this a bit so it's more readable. Also I don't think this will compile. The syntax you want looks more like:

```
if(request.getSession().getAttribute("user").equals("EmilyArroyo") ||
    request.getSession().getAttribute("user").equals("AlexandriaStorm") || ... ){
```

You might also consider shortening it by declaring a `user` variable instead of calling `request.getSession.getAttribute()` a bunch of times.

Reply:
Also, try getting it working with just a single user first, then adding other users to the `if` statement. It's all about starting small and working forward in small steps.